src/main/webapp/views/common/registration.html

```diff
@@ -0,0 +1,51 @@
+<div class="modal fade" id="exampleModalCenter" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
```

Comment:
`aria-labelledby="exampleModalCenterTitle"` There is no id called `exampleModalCenterTitle` in this modal.